### DISCUSSION TITLE
Build is failing because of node_module lint errors

### DISCUSSION THREAD

**Discussion by ValGeorgiev** [callstack/react-native-builder-bob#423] [Q&A] [answered]
### Description

When I run `bob build` the build is failing with the following lint errors from a node_module. It happens only with the following library: `node_modules/fork-ts-checker-webpack-plugin` Is there an option in bob to exclude node_modules when running bob lint checks?

<img width="1190" alt="image" src="https://github.com/callstack/react-native-builder-bob/assets/12056467/f37bbefc-b17b-4e9f-84b7-8fa05eb0cf9f">

### Packages

- [ ] create-react-native-library
- [X] react-native-builder-bob

### Selected options

"react-native-builder-bob": {
    "source": "src",
    "output": "lib",
    "targets": [
      ["commonjs", {"copyFlow": true}],
      "module",
      "typescript"
    ]
  },

### Environment

System:
    OS: macOS 13.4
    CPU: (10) arm64 Apple M1 Pro
    Memory: 601.95 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 16.15.1 - ~/.nvm/versions/node/v16.15.1/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v16.15.1/bin/yarn
    npm: 8.11.0 - ~/.nvm/versions/node/v16.15.1/bin/npm
    Watchman: Not Found
  Managers:
    CocoaPods: 1.12.1 - /Users/valgeorgiev/.rvm/gems/ruby-2.7.5/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 22.2, iOS 16.2, macOS 13.1, tvOS 16.1, watchOS 9.1
    Android SDK: Not Found
  IDEs:
    Android Studio: 2022.2 AI-222.4459.24.2221.10121639
    Xcode: 14.2/14C18 - /usr/bin/xcodebuild
  Languages:
    Java: 18.0.2.1 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.2.0 => 18.2.0 
    react-native: 0.71.4 => 0.71.4 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

**Comment by atlj** [comment, marked as answer]
Can you check this answer: https://stackoverflow.com/a/57653497
I'm also converting this into a discussion since it's not a bug in bob itself.

**Reply by ValGeorgiev** [reply]
Hey @atlj  thanks for the quick response. That was it, i added the tsconfig to the bob typescript configuration and it worked.

```
"react-native-builder-bob": {
    "source": "src",
    "output": "lib",
    "targets": [
      ["commonjs", {"copyFlow": true}],
      "module",
      [
        "typescript",
        {
          "project": "tsconfig.build.json"
        }
      ]
    ]
  },
  ```
